src/write.jl

```diff
@@ -25,6 +25,11 @@ Write JSON.
 ## Keyword Args
 
 * `allow_inf`: Allow writing of `Inf` and `NaN` values (not part of the JSON standard). [default `false`]
+* `inf_mapping`: A function to map `Inf`, `-Inf` and `NaN` values to a custom representation. [default `nothing`]
+
```

Comment:
the extra newlines here feel unnecessary? do we know if they render reasonably?

Reply:
The first extra new line gives better readability in my eyes, making the first line something like a header without leaving the bullet point.
```
    •  inf_mapping: A function to map Inf, -Inf and NaN values to a custom representation. [default nothing]
       if inf_mapping is nothing the mapping is equivalent to
       inf_mapping = x -> x == Inf ? "Infinity" : x == -Inf ? "-Infinity" : "NaN"`
```
vs.
```
    •  inf_mapping: A function to map Inf, -Inf and NaN values to a custom representation. [default nothing] if
       inf_mapping is nothing the mapping is equivalent to inf_mapping = x -> x == Inf ? "Infinity" : x == -Inf ?        
       "-Infinity" : "NaN"`
```
about line 31 I don't care at all. Both versions look reasonable
You decide.

Reply:
Sounds fine to me; it was just unexpected and I wasn't sure if it was just by mistake or looked reasonable.